cmd/simulacra/simulacra.go

```diff
@@ -252,16 +274,9 @@ func main() {
 		log.Default().Fatalf("project and zone must either be non empty or set in GCloud %v", err)
 	}
 
-	options := gce.VMOptions{
-		Platform:    config.Platform,
-		MachineType: agents.RecommendedMachineType(config.Platform),
-		Name:        config.Name,
-		Project:     config.Project,
-		Zone:        config.Zone,
-	}
 	// Create VM Instance.
 	log.Default().Printf("Creating VM Instance, check %s for details", vmInitLogFile)
-	vm, err := gce.CreateInstance(ctx, logger.ToFile(vmInitLogFileName), options)
+	vm, err := createInstance(ctx, &config, logger.ToFile(vmInitLogFileName))
```

Comment:
nit: maybe `config` should already be a `*Config` rather than passing a reference specifically here?